### ISSUE TITLE
Create v1.1.3 tag

### ISSUE THREAD

**Issue by CalumY** [afonasev/flake8-return#126]
* Date you used flake8-return: 28/10/2022
* flake8-return version used, if any: Hoping to use v1.1.3
* Python version, if any: 3.8
* Operating System: WSL

### Description

I was updating our pre-commit hooks to use flake8-return (rather than a forked version), but as the v1.1.3 tag has not been created on Github, the pre-commit hook does not make use of the changes introduced in v1.1.3 on PyPI.

@afonasev, please could you create a tag for v1.1.3 so that pre-commit makes use of the recent updates? 

### What I Did

The abridged pre-commit config used below throws a false positive for R504, which has been fixed in the version released on PyPI.

```bash
repos:
  ...
  - repo: https://github.com/PyCQA/flake8
    rev: 5.0.4
    hooks:
      - id: flake8
        additional_dependencies:
          - flake8-return
          ...
        args: [--config, setup.cfg]
```

**Comment by CalumY** [comment]
@afonasev. Thanks for doing that! Closing as complete.

**Comment by afonasev** [comment]
Thanks for your issue!)

**Comment by CalumY** [comment]
Apologies, I think I closed this prematurely. 

It appears that v1.2.0 is the release that fixes the false positive that we were getting but this version is not currently on PyPI. Would it be possible to publish v1.2.0 on PyPI?

**Comment by afonasev** [comment]
done https://pypi.org/project/flake8-return/1.2.0/

**Comment by CalumY** [comment]
Thank you very much!
